common/src/main/java/org/apache/uniffle/common/netty/protocol/Decoders.java

```diff
@@ -69,6 +70,32 @@ public static ShuffleBlockInfo decodeShuffleBlockInfo(ByteBuf byteBuf) {
         taskAttemptId);
   }
 
+  public static ShufflePartitionedBlock decodeShuffleBlockInfoV1(ByteBuf byteBuf) {
```

Comment:
How about decodeShufflePartitionedBlockV1?

Reply:
@lwllvyb Maybe you miss this comment

Reply:
@maobaolong Fixed.